development/kops/patches/0001-remove-hardcoded-requierment-on-flannel-plugin.patch

```diff
@@ -0,0 +1,24 @@
+From 2ec5afbedada5688c13aa8e8d9ac9ffba3ae4511 Mon Sep 17 00:00:00 2001
+From: Daniel Budris <budris@amazon.com>
+Date: Wed, 3 Aug 2022 11:50:10 -0400
+Subject: [PATCH] remove hardcoded requierment on flannel plugin
+
```

Comment:
Can we add a description here? It would be helpful if we ever have the option of removing this patch with a kops update